yt_dlp/extractor/espn.py

```diff
@@ -283,34 +284,37 @@ def _real_extract(self, url):
 class WatchESPNIE(AdobePassIE):
     _VALID_URL = r'https?://(?:www\.)?espn\.com/(?:watch|espnplus)/player/_/id/(?P<id>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})'
     _TESTS = [{
-        'url': 'https://www.espn.com/watch/player/_/id/ba7d17da-453b-4697-bf92-76a99f61642b',
+        'url': 'https://www.espn.com/watch/player/_/id/dbbc6b1d-c084-4b47-9878-5f13c56ce309',
         'info_dict': {
-            'id': 'ba7d17da-453b-4697-bf92-76a99f61642b',
+            'id': 'dbbc6b1d-c084-4b47-9878-5f13c56ce309',
             'ext': 'mp4',
-            'title': 'Serbia vs. Turkey',
-            'thumbnail': 'https://artwork.api.espn.com/artwork/collections/media/ba7d17da-453b-4697-bf92-76a99f61642b/default?width=640&apikey=1ngjw23osgcis1i1vbj96lmfqs',
+            'title': 'Huddersfield vs. Burnley',
+            'duration': 7500,
+            'thumbnail': 'https://artwork.api.espn.com/artwork/collections/media/dbbc6b1d-c084-4b47-9878-5f13c56ce309/default?width=640&apikey=1ngjw23osgcis1i1vbj96lmfqs',
         },
         'params': {
             'skip_download': True,
         },
     }, {
-        'url': 'https://www.espn.com/watch/player/_/id/4e9b5bd1-4ceb-4482-9d28-1dd5f30d2f34',
+        'url': 'https://www.espn.com/watch/player/_/id/a049a56e-a7ce-477e-aef3-c7e48ef8221c',
         'info_dict': {
-            'id': '4e9b5bd1-4ceb-4482-9d28-1dd5f30d2f34',
+            'id': 'a049a56e-a7ce-477e-aef3-c7e48ef8221c',
             'ext': 'mp4',
-            'title': 'Real Madrid vs. Real Betis (LaLiga)',
+            'title': 'Dynamo Dresden vs. VfB Stuttgart (Round #1) (German Cup)',
+            'duration': 8335,
             'thumbnail': 'https://s.secure.espncdn.com/stitcher/artwork/collections/media/bd1f3d12-0654-47d9-852e-71b85ea695c7/16x9.jpg?timestamp=202201112217&showBadge=true&cb=12&package=ESPN_PLUS',
         },
         'params': {
             'skip_download': True,
         },
     }, {
-        'url': 'https://www.espn.com/espnplus/player/_/id/317f5fd1-c78a-4ebe-824a-129e0d348421',
+        'url': 'https://www.espn.com/watch/player/_/id/47950544-b39f-4e47-a982-b2132d20c8d1',
```

Comment:
this is supposed to test `espnplus/` URL

Reply:
My mistake, reverted